docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -1 +1,78 @@
 = Glossary of terms related to {ProductName}
+:icons: font
+:source-highlighter: highlightjs
+
+**{ProductName}** +
+A platform to automate the process of building, testing, and releasing applications. {ProductName} offers enterprise-grade security and customizable feature sets.   
+
+**build pipeline customization** +
+The ability to update and manage build pipelines for each component in an application. 
+
+**cluster** +
+A Kubernetes deployment with nodes that run containerized applications and a control plane that manages the nodes.
+
+**component** +
+An internal representation of a container image that {ProductName} builds and tests, using source code from a git repository. 
+
+**enterprise contract (EC)** +
+A set of release policies that you apply to your application snapshot. You can use the EC to prevent releases that are not compliant with Supply-chain Levels for Software Artifacts (SLSA) guidelines. 
+
+**integration test** +
+A pipeline that you set up in GitHub. When you add components, it tests each one individually, and then tests the application as a whole.
+
+**IntegrationTestScenario (ITS)** +
+A Kubernetes resource that defines a pipeline to run. The integration service uses the ITS to test application components.
+
+**managed workspace** +
+A workspace that mirrors some details of an already existing workspace. Supporting teams can create a managed workspace to grant limited permissions to development teams in the production environment. 
+
+**persister** +
+A component that moves all of the relevant PipelineRun information, known as the Pipeline output, to an external storage that is outside of the cluster’s etcd database. The persister runs after the system completes the PipelineRun.
+
+**pipelines as code** +
+A practice that defines pipelines by using source code in Git. Pipelines as Code is also the name of link:https://pipelinesascode.com[a subsystem] that executes those pipelines.
+
+**pipeline results** +
+Systems that retain the history and details of builds. 
+
+**PipelineRun** +
+A collection of TaskRuns that are arranged in a specific order of execution. 
+
+**provenance** +
+{ProductName} produces SLSA provenance, which consists of the attestation for an artifact, and a signature for that attestation. Attestation lists the steps that {ProductName} took to create a given artifact. The signature enables you to verify that no one tampered with that attestation.  
```

Comment:
The provenance lists the steps

Reply:
Isn't attestation the specific component of provenance that lists the steps? That's how we defined it [here](https://konflux-ci.dev/docs/Konflux-CI/discover/slsa-conformity/#slsa-provenance).

Or would you like me to change this to say:
"Konflux produces SLSA provenance, which includes a list of steps that Konflux took to build a given artifact, and a signature to verify that the provenance comes from Konflux."

Reply:
An attestation is generically [a claim being made about a subject](https://github.com/in-toto/attestation?tab=readme-ov-file#in-toto-attestation-framework) (i.e. the container that is produced). The SLSA provenance is a _type_ of attestation but there are many different types of attestations ... some that have been [defined by the community](https://github.com/in-toto/attestation/tree/main/spec/predicates), but others can certainly exist even without being vetted by the in-toto maintainers.

If we are talking about the build steps, that is the provenance attestation .. both here and in https://konflux-ci.dev/docs/#slsa-provenance. I didn't catch that issue in the overview previously.

Reply:
Ok, I updated this def and the SLSA overview doc accordingly.